contracts/src/PDPService.sol

```diff
@@ -343,6 +379,40 @@ contract PDPService {
         bytes32[] proof;
     }
 
+    // Roll over to the next proving period
+    // 1. Resample challenge
+    // 2. Track newly added roots for proving
+    // 3. Remove scheduled removals
```

Comment:
Expand this documentation to address the caller (SP). Explain that it processes removals and expands the challenge range. Explain also that it can be called without submitting a proof beforehand. But that listener contracts may consider this as some kind of failure.